props/saber_fett263_buttons.h

```diff
@@ -251,10 +251,15 @@ SaberFett263Buttons() : PropBase() {}
 
       // EVENT_PUSH
       if (fabs(mss.x) < 3.0 &&
-          mss.y * mss.y + mss.z * mss.z > 120 &&
+          mss.y * mss.y + mss.z * mss.z > 100 &&
           fusor.swing_speed() < 30 &&
           fabs(fusor.gyro().x) < 10) {
-        Event(BUTTON_NONE, EVENT_PUSH);
+        if (millis() - push_begin_millis_ > 10) {
```

Comment:
At 10 ms, this loop will only execute a few times, and it might not be worth doing this extra step.
Wish we could test for "speed" instead of "mss", that would make all of this much better.

Reply:
Did we try speed?  It's working for me currently but I can test another method, with the 25 it was inconsistent because the gesture is meant to be a short movement so I had to lower to have it work.  but if we can improve I'm open to trying other options.